code/style.css

```diff
@@ -1,7 +1,483 @@
+  border-radius: 1rem;
+  box-shadow: 0 5px 8px 0 rgba(0,0,0,0.35);
+}
+
+.form-input-container {
+  padding: 20px;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.form-heading h3 {
+  font-family: 'Montserrat', sans-serif;
+  color: #495867;
+  margin: 5px 0 36px 0;
+  text-align: center;
+}
+
+.form-group {
+  margin-bottom: 15px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: #495867;
+}
+
+.form-group label {
+  width: 100px;
+  margin-right: 5px;
+  color: #495867;
+}
+
+.form-group input {
+  border-top-style: hidden;
+  border-right-style: hidden;
+  border-left-style: hidden;
+  background: rgba(0, 0, 0, 0);
+  color: #495867;
+}
+
+.form-group input,
+.form-group textarea,
+.radio-button {
+  width: 200px;
+}
+
+.number-group {
+  display: flex;
+  justify-content: space-between;
+  width: 305px;
+  margin-bottom: 15px;
+  color: #495867;
+  font-size: 16px;
+}
+
+.number-group input {
+  width: 50px;
+  text-align: center;
+  color: #495867;
+  border: 1px solid #495867;
+}
+
+.radio-group {
+  display: flex;
+  flex-direction: row;
+  margin-bottom: 15px;
+  color: #495867;
+}
+
+.radio-group p {
+  width: 100px;
+  margin-right: 5px;
+}
+
+.radio-button {
+  font-size: 12px;
+}
+
+.radio-button label {
+  margin-right: 3px;
+}
+
+.checkbox-group {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 15px;
+  color: #495867;
+  font-size: 12px;
+  width: 305px;
+}
+
+.checkbox-group-terms,
+.checkbox-group-order {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 10px;
+}
+
+.checkbox-group label {
+  width: 250px;
+}
+
+.checkbox-group-news {
+  display: flex;
+  justify-content: space-between;
 }
+
+.form-submit {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.form-submit-button {
+  background-color: #495867;
+  color: #E7AD99;
+  padding: 8px 18px;
+  text-align: center;
+  text-transform: uppercase;
+  text-decoration: none;
+  font-size: 16px;
+  cursor: pointer;
+  border: none;
+  border-radius: 10px;
+}
+
+@media (max-width:768px) {
+  html {
+    font-size: 14px;
+  }
+}
+
+@media (max-width: 600px) {
+
+  header {
+    padding: 0;
+  }
+
+  header .navbar {
+    justify-content: space-around;
+  }
+
+  .navbar h2 {
+    margin: 0 10px;
+    font-size: 18px;
+  }
+
+  .hamburger {
+    margin-right: 5px;
+  }
+
+  .navbar img {
+    margin-left: 5px;
+  }
+
+  .hero-content {
+    padding-top: 105px;
+    height: 70vh;
+  }
+
+  .hero-headline {
+    font-size: 36px;
+  }
+
+  .hero-video-playing {
+    display: none;
+  }
+
+  /* The image in the picture tag doesn't seem to show. only the img tag is showing. I can't figure out why. */
+  .hero-picture {
+    display: flex;
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    z-index: 2;
+    object-fit: cover;
+    object-position: top center;
+  }
+
+  .hero-picture img {
+    width: 107%;
```

Comment:
How come the width is more than 100% or is it a typo? Either way, it makes the header stick out when testing on smaller devices so please give this a check!